Resources/Prototypes/_Floof/Entities/Mobs/NPCs/playerreadymobs.yml

```
@@ -0,0 +1,119 @@
+- type: entity
+  id: BaseFriendlyPlayerMob
+  parent: StripableInventoryBase
+  abstract: true
+  categories: [ HideSpawnMenu ]
+  components:
+    - type: Hands
+    - type: ComplexInteraction
+    - type: Inventory
+      templateId: nakedcrew
+    - type: MailReceiver
+    - type: NpcFactionMember
+      factions:
+      - NanoTrasen
+    - type: Puller
+      needsHands: false
```

You are giving them hands, so they should require hands to pull
```suggestion
      needsHands: true
```